workspaces/liferay-sample-workspace/language/README.markdown

```diff
@@ -0,0 +1,7 @@
+To add or override translations, place your entries inside the `lang/Language.properties` file. After making changes, run the command `../gradlew buildLang` to generate the default translation files. This command will create copies of all keys and values from your `Language.properties` file into the generated files.
+
+Every time you modify the `Language.properties` file, you should run `../gradlew buildLang` to update the translation files accordingly. Any changes you make directly in the generated translation files will be preserved.
+
+### Important
+
+The translations import will fail if the developer feature flag `LPD-27222` is not enabled.
```

Comment:
@thiago-buarqque Maybe we just keep it very simple. 

```
To add or override translations, place your translations inside the `lang/Language.properties` file.

To generate translations for other locales using Liferay's Language Builder tool run `../gradlew buildLang` from the `lang` directory.

When the translations are deployed to a Liferay instance, they will be imported into the Language Override widget.
```

@rbohl what do you think? Any help would be much appreciated!

Reply:
Some references: 
- https://learn.liferay.com/w/dxp/liferay-development/liferay-internals/extending-liferay/customizing-localization/generating-translations-automatically?p_l_back_url=%2Fsearch%3Fq%3DbuildLang&p_l_back_url_title=Search&highlight=buildLang
- https://learn.liferay.com/w/dxp/system-administration/configuring-liferay/changing-translations-with-language-override?p_l_back_url=%2Fsearch%3Fq%3Dlanguage%2Boverride&p_l_back_url_title=Search&highlight=language%20override

Reply:
@ethib137 I am considering you are going to rely on mentioning the feature flag in the official docs, right?

Reply:
@rbohl I like your suggestion, but I see we have a redundancy:

```markdown
You can import translations into Liferay's Language Override tool. [...]

[...]

When deployed to a Liferay instance, translations are imported into the Language Override tool.
```

Along with other small changes, my suggestion is:

```markdown
To import translations into Liferay's Language Override tool, place them into a `[Workspace Root]/language/lang/Language.properties` file using `key=value` syntax.

To generate translations for other locales, use Liferay's Language Builder tool. Run `../gradlew buildLang` from the `language` directory. This task automatically runs when you deploy the translations with `../gradlew deploy`.
```

Reply:
Thiago and I came up with this as our suggestion, let us know what you think @ethib137:

> To import translations into Liferay's Language Override tool, place them into a `[Workspace Root]/language/lang/Language.properties` file using `key=value` syntax and run `../gradlew deploy` from the `language` folder.
> 
> To generate translations for other locales, use Liferay's Language Builder tool. Run `../gradlew buildLang` from the `language` directory. This task creates locale-specific files (e.g., `Language_pt_BR.properties`) with automatic copies of the keys and values from the `Language.properties` file. You can then translate the values and deploy the translations. The `buildLang` task runs automatically when you deploy the translations, but values with _(Automatic Copy)_ at the end are not used in Liferay. You must include translated properties in the generated files for them to be effective.
> 
> Related documentation:
> * [Generating Translations Automatically](https://learn.liferay.com/w/dxp/liferay-development/liferay-internals/extending-liferay/customizing-localization/generating-translations-automatically)
> * [Changing Translations with Language Override](https://learn.liferay.com/w/dxp/system-administration/configuring-liferay/changing-translations-with-language-override)

Reply:
This looks pretty good to me. We just have to see what BChan thinks.